plugins/gcp/resoto_plugin_gcp/resources/sqladmin.py

```diff
@@ -619,6 +628,17 @@ def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
             if cert.self_link:
                 builder.add_edge(self, reverse=True, clazz=GcpSslCertificate, link=cert.self_link)
 
+    def post_process(self, graph_builder: GraphBuilder, source: Json) -> None:
+        for cls in [GcpSqlBackupRun, GcpSqlDatabase, GcpSqlUser]:
+            if spec := cls.api_spec:
+                try:
+                    items = graph_builder.client.list(spec, instance=self.name, project=self.instance_project)
+                    cls.collect(items, graph_builder)
+                except Exception as e:
+                    msg = f"Error while collecting {cls.__name__}: {e}"
+                    graph_builder.core_feedback.info(msg, log)
```

Comment:
Than the base needs to change as well. Otherwise we will not have any consistent error handling.
I would opt to put error handling into the client and leave it out of the resource.

Reply:
Alright, I took it out of the resource